swan/scripts/03_userconfig.sh

```diff
@@ -0,0 +1,122 @@
+#!/bin/bash
+
+# Author: Danilo Piparo, Enric Tejedor, Pedro Maximino, Diogo Castro 2023
+# Copyright CERN
+
+_log () {
```

Comment:
This is not accessible?

Reply:
It is not accessible no, I had to redefine it